Document XLS-80 Permissioned Domains (#124)
ripple/opensource.ripple.com: docs/xls-80d-permissioned-domains/permissioneddomainset.md

| `AcceptedCredentials` | Array             | Array             | Yes       | A list of 1 to 10 [**Accepted Credentials objects**](#accepted-credentials-objects) that grant access to this Domain. The list does not need to be sorted, but it cannot contain duplicates. When modifying an existing domain, this list replaces the existing list. |

{% raw-partial file="/docs/_snippets/accepted-credentials-objects.md" /%}

## {% $frontmatter.seo.title %} Flags

There are no flags defined for {% $frontmatter.seo.title %} transactions.

## Error Cases

Besides errors that can occur for all transactions, {% $frontmatter.seo.title %} transactions can result in the following [transaction result codes](https://xrpl.org/docs/references/protocol/transactions/transaction-results):

| Error Code                | Description |
|:--------------------------|:------------|
| `tecDIR_FULL`             | The transaction would create a new PermissionedDomain, but the sender's owner directory is full. |
| `tecINSUFFICIENT_RESERVE` | The transaction would create a new PermissionedDomain, but the sender does not have enough XRP to meet the increased owner reserve. |

> | `tecNO_ENTRY`             | The transaction attempted to modify an existing Domain that does not exist. Check the `DomainID` field of the transaction. |

Reviewer: ```suggestion
| `tecNO_ENTRY`             | The transaction attempted to modify a Domain that does not exist. Check the `DomainID` field of the transaction. |
```
